rules/cloud/okta/okta_admin_role_assignment_created.yml

```diff
@@ -0,0 +1,21 @@
+title: Okta Admin Role Assignment Created
+id: 139bdd4b-9cd7-49ba-a2f4-744d0a8f5d8c
+status: test
+description: Detects when a new admin role assignment is created
```

Comment:
Would be helpful if you could include more in the description. For example, say something like. You could modify it as you know this better than me.
```suggestion
description: `Detects when a new admin role assignment is created. Which could be a sign of privilege escalation or persistence`
```